sentry-samples/sentry-samples-netflix-dgs/src/main/java/io/sentry/samples/netflix/dgs/ActorsDataloader.java

```diff
@@ -0,0 +1,30 @@
+package io.sentry.samples.netflix.dgs;
+
+import com.netflix.graphql.dgs.DgsDataLoader;
+import io.sentry.samples.netflix.dgs.graphql.types.Actor;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import org.dataloader.MappedBatchLoader;
+import org.jetbrains.annotations.NotNull;
+
+@DgsDataLoader(name = "actors")
+public class ActorsDataloader implements MappedBatchLoader<Integer, Actor> {
+
+  @Override
+  public CompletionStage<Map<Integer, Actor>> load(Set<Integer> keys) {
+    return CompletableFuture.supplyAsync(
+        () -> {
+          final @NotNull Map<Integer, Actor> map = new HashMap<>();
+          for (Integer key : keys) {
+            if (key != null && key == -1) {
+              throw new RuntimeException("Causing an error while loading actor");
```

Comment:
Why do we throw an exception here? how is it handled? is it going to break the app?